src/components/common/EnrollmentStatus.js

```diff
@@ -8,9 +8,9 @@ import { reverse } from 'console/urls';
 
 // Ideally the prop.recipe would be an Immutable Map, but Ant's Table works with
 // plain JS objects, which means this component can not be Pure.
-export default class EnrollmentSatus extends React.Component {
+export default class EnrollmentStatus extends React.Component {
   static propTypes = {
-    recipe: PropTypes.object.isRequired,
+    currentRevision: PropTypes.object.isRequired,
```

Comment:
At first I thought; this should be called "revision" because `EnrollmentStatus` doesn't care or need to know what kind of revision it is. hashtag purist. But I guess it's actually quite pleasant to know that when reading the code.